src/main/java/seedu/address/model/task/Link.java

```diff
@@ -3,6 +3,7 @@
 import static java.util.Objects.requireNonNull;
 
 public class Link {
+    public static final String INVALID_LINK = "The link provided is not a valid URL!";
```

Comment:
A suggestion, maybe can put this error message under "seedu.address.commons.core.Messages" instead. Then ins ParserUtil can import the message

Reply:
I'm following how these messages are implemented for the objects used in `Person` class.

Updated to `MESSAGE_CONSTRAINTS` instead.